android-versions.md

```diff
@@ -25,6 +25,6 @@ color: #f4de50
 |8.0|August 21, 2017|Oreo|Notification improvements, Multi-display support, Wi-Fi Assistant |
 |9.0|August 6, 2018|Pie|Rounded corners across the UI, Android Dashboard, Redesigned volume slider |
 |10|September 3, 2019|Q|New safeguards to protect user privacy that need to support in our app. |
-|11|September 8, 2020|R|Conversations, bubbles, device and media controls, 5G, biometric and more.|
+|11|September 8, 2020|Red Velvet Cake|Conversations, bubbles, device and media controls, 5G, biometric and more.|
```

Comment:
Please raise PR with one cheat sheet changes at a time

Reply:
Since I am a newbie, I don't know how to do that.
All pr gets automatically merged to one.